src/for-redux-swr/index.ts

```diff
@@ -0,0 +1 @@
+export {SpreadoSetupBetweenReduxSwr} from './SpreadoSetupImpl';
```

Comment:
nit: Exporting all would just be fine.

```suggestion
export * from './SpreadoSetupImpl';
```